MeesGame/MeesGame.csproj

```diff
@@ -227,6 +230,7 @@
     <Content Include="Content\levels\05 all%401ce.lvl">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
+    <Content Include="Gameplay\Content\winScreenOverlay.png" />
```

Comment:
This PR shouldn't have changed the csproj file, if this line needs to be added, do that in the other PR.